contracts/merkle-distributor/MerkleDistributor.sol

```diff
@@ -193,6 +193,15 @@ contract MerkleDistributor is Ownable {
         return claimedWord & mask == mask;
     }
 
+    /**
+     * @notice Returns rewardToken set by admin for windowIndex.
+     * @param windowIndex merkle root to check.
+     * @return address Reward token address
+     */
+    function getRewardTokenForWindow(uint256 windowIndex) public view returns (address) {
```

Comment:
Adding for usefulness in https://github.com/across-protocol/across-token/pull/32/files